sdk/core/azure-core-test/src/main/java/com/azure/core/test/implementation/models/HttpBinFormDataJson.java

```diff
@@ -0,0 +1,271 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
nit for this and the other files in this folder: can we figure out how to make it a rename instead of a delete/add? I dunno if the source history for this is useful or not but defaulting to keeping it would be good.